sdk/communication/Azure.Communication.Common/tests/Pipeline/HMACAuthenticationPolicyTests.cs

```diff
@@ -22,13 +22,13 @@ public async Task TestHMACPolicyProcess()
             await SendGetRequest(transport, authPolicy);
             var headers = transport.SingleRequest.Headers;
 
-            Assert.True(headers.Contains("Date"));
+            Assert.True(headers.Contains("x-ms-date"));
             Assert.True(headers.Contains("Authorization"));
 
             Assert.True(headers.TryGetValue("x-ms-content-sha256", out var shaValue));
             Assert.AreEqual(expectedShaValue, shaValue);
 
-            var expectedAuthHeader = "HMAC-SHA256 SignedHeaders=date;host;x-ms-content-sha256";
```

Comment:
Same comment here

Reply:
I think we have to change here since Date is no longer part of the auth header